### DISCUSSION TITLE
已下载内容不显示

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#507] [bug]
### Bug 描述

之前修改过下载存放目录，升级4.6.26后，下载完成里显示内容为0。

### 操作步骤

```plain text
临时解决方法：先打开下载完成界面，再去设置点击修改存放目录，然后取消。回到下载页就可以正常显示，但仅本次有效。
重启软件后失效，需重复此操作。

**Comment by ywmoyue** [comment]
问下你这边总共有多少下载视频，我改了下加载下载视频的步骤现在会在读取完成后全部显示出来而不是和之前一样一边显示一边加载

如果视频很多需要读取很久的话体验上会比之前版本差，下个版本我改进一下

**Reply by lyppiao** [reply]
205个视频，加起来22G。

**Reply by ywmoyue** [reply]
@lyppiao 麻烦验证一下 https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.26.1

**Reply by lyppiao** [reply]
可以了，一打开软件就会在后台加载，且速度大幅度提高。我这么多视频，25秒就全部加载完成了，上个版本几分钟都出不来。
